src/opnsense/service/conf/actions.d/actions_dns.conf

```
@@ -3,6 +3,7 @@ command:/usr/local/sbin/pluginctl -c dns_reload
 parameters:
 type:script_output
 message:rewriting /etc/resolv.conf and /etc/hosts
+description:Regenerate fixed hostnames (/etc/hosts) and nameservers (/etc/resolv.conf)
```

Can we simplify this a bit perhaps like the following:

```suggestion
description:Reload local DNS configuration
```

---

I feared it might be too long -- my concern with 'local DNS' is it might be confused with reloading unbound or dnsmasq.

Does `Reload local DNS resolver configuration` work?

---

Perhaps it’s best to use the message approach and say rewrite xxx and yyy explicitly? Though the problem is it could change and it has before. The job was always called reload_dns internally but it’s more of an apply on a system level collecting from all plugged interface configurations.